src/main/java/org/spongepowered/common/mixin/core/item/MixinItemFishingRod.java

```diff
@@ -54,10 +54,12 @@ public ItemStack onItemRightClick(ItemStack itemStack, World world, EntityPlayer
             player.fishEntity.handleHookRetraction();
         } else {
             EntityFishHook fishHook = new EntityFishHook(world, player);
-            if (!Sponge.getGame().getEventManager()
+            if (world.isRemote || !Sponge.getGame().getEventManager()
```

Comment:
@Zidane 
> Introduction of another enum is un-necessary. The one available in Platform would suffice.

How do you handle `BOTH` or `DEFAULT`? Values in annotations can't be set to `null`.

> Scratch bullet 3, don't want to go down the route of "Side Only classes" at this time.

Sounds good. If someone wants to do something like that, either something on the `@Plugin` annotation would suffice, or they can register the listener only on the client.
```java
if (game.getPlatform().getType() == Platform.Type.CLIENT) { 
    game.getEventManager().register(this, new ClientSideListener());
}
```